tests/storage/storage.go

```diff
@@ -315,8 +318,10 @@ var _ = SIGDescribe("Storage", func() {
 					libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 				}
 			},
-				Entry("[test_id:3132]with Disk PVC", newRandomVMIWithPVC),
-				Entry("[test_id:3133]with CDRom PVC", newRandomVMIWithCDRom),
+				Entry("[test_id:3132]with Disk PVC", newRandomVMIWithPVC, nil),
+				Entry("[test_id:3133]with CDRom PVC", newRandomVMIWithCDRom, nil),
+				Entry("with Disk PVC and terminationGracePeriod = 0", newRandomVMIWithPVC, pointer.P(int64(0))),
+				Entry("with CDRom PVC and terminationGracePeriod = 0", newRandomVMIWithCDRom, pointer.P(int64(0))),
```

Comment:
Let's move this one step forward. The `newRandomVMIWithPVC` and `newRandomVMIWithCDRom` already have a `libvmi` equivalent. We can just do:
libvmi.New(
					libvmi.WithPersistentVolumeClaim("disk0", claimName),
					libvmi.WithResourceMemory("256Mi"),
					libvmi.WithRng())
```
Entry("[test_id:3132]with Disk PVC", libvmi.New(
					libvmi.WithPersistentVolumeClaim("disk0", claimName),
					libvmi.WithResourceMemory("256Mi"),
					libvmi.WithRng())),
Entry("with Disk PVC and terminationGracePeriod = 0", libvmi.New(
					libvmi.WithPersistentVolumeClaim("disk0", claimName),
					libvmi.WithResourceMemory("256Mi"),
					libvmi.WithRng(),
					libvmi.WithTerminationGracePeriod(1))),
```

Reply:
`claimName` is just a parameter for:
`newRandomVMIWithPVC := func(claimName string) *v1.VirtualMachineInstance {...}  `.
I fear it will become less readable.

Reply:
That is true but here we don't seem to change it. Therefore I would just inline it.
Why do you think it would be less readable?

Reply:
for instance now `libvmi.WithResourceMemory("256Mi")` is defined only once in `newRandomVMIWithPVC` function and then reused in many different tests: with a single line change we can address all the involved tests while if we inline it everywhere, we will have to touch many different lines in the case we will decide to require more memory for the VMIs.